File: Funproject/Funproject/CoreData/NSManagedObjects/Company+CoreDataProperties.m
//
//  Company+CoreDataProperties.m
//  Funproject
//
//  Created by Stig von der Ahé on 02/10/2024.
//
//

#import "Company+CoreDataProperties.h"

@implementation Company (CoreDataProperties)

+ (NSFetchRequest<Company *> *)fetchRequest {
	return [NSFetchRequest fetchRequestWithEntityName:@"Company"];
}

@dynamic name;
@dynamic employees;

@end

File: Funproject/Funproject/CoreData/NSManagedObjects/Company+CoreDataProperties.swift
//
//  Company+CoreDataProperties.swift
//  Funproject
//
//  Created by Stig von der Ahé on 02/10/2024.
//
//

import Foundation
import CoreData


extension Company {

    @nonobjc public class func fetchRequest() -> NSFetchRequest<Company> {
        return NSFetchRequest<Company>(entityName: "Company")
    }

    @NSManaged public var name: String?
    @NSManaged public var employees: Employee?

}
